perun.go

```diff
@@ -253,7 +253,7 @@ type (
 	ChProposalNotif struct {
 		ProposalID string
 		Currency   string
-		Proposal   *pclient.ChannelProposal
+		Proposal   pclient.ChannelProposal
```

Comment:
Since this interface now has a `Proposal` method that retrieves the underlying Proposal struct, is it possible to rename this, so that we don't use `Proposal.Proposal()` ?

Reply:
Yes, I have changed `Proposal` to `ChProposal` which is addressed in this [commit](https://github.com/hyperledger-labs/perun-node/pull/110/commits/419c09f1a62eb87b03be5594f43da29dfd7ed31c)

Reply:
Can you move that change from [add transactor interface](https://github.com/hyperledger-labs/perun-node/pull/110/commits/419c09f1a62eb87b03be5594f43da29dfd7ed31c) commit to this one, as it is a part of this change set ?

Reply:
[Updated](https://github.com/hyperledger-labs/perun-node/pull/110/commits/03c61bdc7d403de9979c2b71a061e4521870b65a)